test/gtest/common/test_helpers.h

```diff
@@ -72,6 +76,25 @@ int test_time_multiplier();
 void safe_usleep(double usec);
 
 
+/**
+ * Print the address for the given interface name
+ */
+void print_ip(char *if_name, struct sockaddr *ifa_addr);
+
+
+/**
+ * Check if the given interface is an Infiniband interface and if there is an
+ * IPv4 address on it.
+ */
+bool is_iface_ipoib(struct ifaddrs *ifa);
+
+
+/**
+ * From the given list of available interfaces, set addr to the first IPoIB
+ * address present. If none is present, set addr to NULL.
+ */
+void set_ip(struct ifaddrs **ifaddr, const struct sockaddr** addr);
```

Comment:
the function name is not related at all to what the function does
it should be
`struct sockaddr* find_ipoib_addr(struct ifaddrs **ifaddrs);`